packages/compass-e2e-tests/helpers/compass.ts

```diff
@@ -646,6 +644,11 @@ async function startCompassElectron(
         binary: maybeWrappedBinary,
         args: chromeArgs,
       },
+      // from https://github.com/webdriverio-community/wdio-electron-service/blob/32457f60382cb4970c37c7f0a19f2907aaa32443/packages/wdio-electron-service/src/launcher.ts#L102
+      'wdio:enforceWebDriverClassic': true,
+    },
+    'wdio:chromedriverOptions': {
+      // TODO: enable logging so we don't have to debug things blindly
```

Comment:
Maybe I'm just really bad at finding undocumented options, but I would be eternally grateful to whoever gets us more logs because I just lost hours of the app silently and cryptically stopping on startup before ever showing a window.